js/modules/data.src.js

```diff
@@ -412,10 +412,60 @@ Highcharts.ajax = function (attr) {
  * @apioption data.table
  */
 
+/**
+ * A URL to a remote CSV dataset.
+ * Will be fetched when the chart is created using Ajax.
+ *
+ * @type {String}
+ * @apioption data.csvURL
+ */
+
+/**
+ * A URL to a remote JSON dataset, structured as a row array.
+ * Will be fetched when the chart is created using Ajax.
+ *
+ * @type {String}
+ * @apioption data.rowsURL
+ */
+
+/**
+ * A URL to a remote JSON dataset, structured as a column array.
+ * Will be fetched when the chart is created using Ajax.
+ *
+ * @type {String}
+ * @apioption data.columnsURL
+ */
+
+/**
+ * Sets the refresh rate for data polling when importing remote dataset by 
+ * setting [data.csvURL](data.csvURL), [data.rowsURL](data.rowsURL), or 
+ * [data.columnsURL](data.columnsURL).
+ *
+ * Note that polling must be enabled by setting 
+ * [data.enablePolling](data.enablePolling) to true.
+ *
+ * The value is the number of seconds between pollings.
+ * It cannot be set to less than 1 second.
+ *
+ * @default 1
+ * @type {Number}
+ * @apioption data.dataRefreshRate
+ */
+
+/**
+ * Enables automatic refetching of remote datasets every _n_ seconds (defined by
+ * setting [data.dataRefreshRate](data.dataRefreshRate)).
+ *
+ * Only works when either 
```

Comment:
Only works when either ....

Reply:
Whops, I probably managed to delete the rest of the sentence in a VIM spasm.  Fixed in ae0ab95